pontoon/contributors/static/js/settings.js

```diff
@@ -1,4 +1,118 @@
 $(function () {
+  $('.field .input')
+    .unbind('keydown.pontoon')
+    .bind('keydown.pontoon', function (e) {
+      if (e.which === 13) {
+        $(this).trigger('blur');
+        return false;
+      }
+    });
+
+  $('.field .input').on('blur', function (e) {
+    e.preventDefault();
+
+    const self = $(this);
+    const value = self.val().trim();
+    const attribute = self.data('attribute');
+    const originalValue = self.data('original-value') || '';
+
+    if (value === originalValue) {
+      return;
+    }
+
+    $.ajax({
+      url:
+        '/user/' +
+        $('#profile input[name="username"]').val() +
+        '/attributes/field/',
```

Comment:
This is duplicated often enough that it'd be a good idea to add a utility function for it, something like:
```js
function userAttributeUrl(action) {
  const username = $('#profile input[name="username"]').val();
  return `/user/${username}/attributes/${action}/`;
}
```